rfcs/20190802-model-garden-redesign.md

```diff
@@ -0,0 +1,207 @@
+# TensorFlow Official Model Garden Redesign
+
+| Status        | Proposed       |
+:-------------- |:---------------------------------------------------- |
+| **Author(s)** | Jing Li (jingli@google.com), Hongkun Yu (hongkuny@google.com), Xiaodan Song (xiaodansong@google.com) |
+| **Sponsor**   | Edd Wilder-James (ewj@google.com)                 |
+| **Updated**   | 2019-08-02                                           |
+
+## Objective
+
+This document presents a proposal to redesign TensorFlow official model garden.
+We aim to provide a central and reliable place to contain popular examples,
+state-of-the-art models and tutorials to demonstrate the best practice in TF2.0
+and illustrate real-world use cases.
+
+## Motivation
+
+The current [TF official model garden](https://github.com/tensorflow/models/tree/master/official)
```

Comment:
I have this same question.

Reply:
The current plan is still to keep the model garden lie in the current directory. But we are going to work with TF Hub to provide a unified UI to provide both pretrained models and links to model codes, hopefully it will make it easier for users to find. Thanks!

Reply:
I am not sure what others' thoughts, but to me, *model garden directory* is `https://github.com/tensorflow/models/tree/master`, while the *official* means the `official` sub directory...